code/modules/reagents/chemistry/reagents/drugs.dm

```diff
@@ -670,29 +751,41 @@
 	addiction_chance = 10
 	addiction_threshold = 5
 	metabolization_rate = 0.6
+	addiction_decay_rate = 0.1 //very low to force them to take time off of meth
 	taste_description = "wiper fluid"
 
+/datum/reagent/lube/ultra/on_mob_add(mob/living/L)
+	ADD_TRAIT(L, TRAIT_GOTTAGOFAST, id)
+	if(ishuman(L))
+		var/mob/living/carbon/human/H = L
+		H.physiology.stun_mod *= 1.5
```

Comment:
var

Reply:
Ah apologies, this code is exactly the same as meth so i thought it was commenting on that.